lib/tasks/create_tables.rake

```diff
@@ -0,0 +1,75 @@
+namespace :db do
+  desc 'Create all report schema tables'
+  task :create_tables do
+    include Report::MysqlConnection
+
+    create_hits_by_day reports_client
+    create_orders_by_day reports_client
+    create_import_logs reports_client
+  end
+end
+
+def create_hits_by_day(client)
+    query =<<EOF
+CREATE TABLE reports.hits_by_day (
+  id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  created_at date DEFAULT NULL,
+  tracker_name varchar(70) CHARACTER SET utf8 DEFAULT NULL,
+  destination_name varchar(255) CHARACTER SET utf8 DEFAULT NULL,
+  hits INTEGER NOT NULL DEFAULT -1
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+EOF
+    client.query query
+
+    query = "CREATE INDEX reports_hits_by_day_created_at_idx ON reports.hits_by_day(created_at) USING BTREE;"
```

Comment:
I'd rather keep this query into a separated file under /sql folder. It is easy to maintain and change. You do not have to do it now. But keep it in mind for a future refactor.